src/common/snippets/src/lowered/pass/serialize_control_flow.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/lowered/pass/serialize_control_flow.hpp"
+
+#include "openvino/pass/serialize.hpp"
+#include "snippets/itt.hpp"
+#include "snippets/lowered/linear_ir.hpp"
+#include "snippets/op/serialization_node.hpp"
+#include "snippets/snippets_isa.hpp"
+
+namespace ov {
+namespace snippets {
+namespace lowered {
+namespace pass {
+
+SerializeControlFlow::SerializeControlFlow(const std::string& xml_path, const std::string& bin_path)
+    : m_xml_path(xml_path),
+      m_bin_path(bin_path) {}
```

Comment:
bin_path can always be reconstructed from the xml_path, can't it? For example if xml_path is `/huba/bu/ba.xml`, then bin_path must always be `/huba/bu/ba.bin`. Have you seen any use cases when the bin file is stored somewhere else?
I mean, that potentially we can take only one arg `name` and create both xml_path and bin_path from it.

Reply:
I'd suggest to just set "/pth/name.xml" (not only `name`) because it seems more readable and logically for me at least.
Sometimes I want to save IR to a directory. I'd prefer to see  `serialize(/path/name.xml)` instead of  `serialize(/path/name)`, for example.

Reply:
Thanks for the suggestions! I left only one argument in serialization pass constructors: path to the xml file